cfpb/cfgov-refresh
# Notification: convert explanation to TextBlock and remove safe filter

## Changes

- Convert notification explanation to a TextBlock to make input easier.
- Remove `| safe` filter from explanation since it is no longer a rich text input.

## Testing

1. Create a new sublanding page and add a notification. If you add more than one line to explanation the text area should grow to accommodate the text.
2. Add a new browse page and add a conference registration form.
3. Fill in the required fields. Add a link to the success message Rich Text Field. 
4. Publish the form, fill in the details  and hit the "register" button and see that the success notification has a notification with HTML:
 
<img width="703" alt="Screen Shot 2019-03-12 at 5 31 17 PM" src="https://user-images.githubusercontent.com/704760/54237593-b029a280-44ec-11e9-867b-0006ebb91261.png">
